src/js/i18n.js

```diff
@@ -102,13 +167,24 @@
    * attribute and replacing the innerHTML with the appropriate translation.
    */
   function load_translations() {
+    let lang = document.documentElement.lang || "en";
+    // Get all data-i18n attributes on el
     document.querySelectorAll("[data-i18n]").forEach(function (el) {
-      var key = el.getAttribute("data-i18n");
-      var lang = document.documentElement.lang || "en";
-      if (!STRINGS[key]) {
-        console.error("Missing translation for " + key);
-      }
-      el.innerHTML = STRINGS[key][lang];
+      // get all translations keys on the el
+      let keys = el.getAttribute("data-i18n").split(" ");
+      keys.forEach((key) => {
+        // Check if the translation is missing, log an error
+        if (!STRINGS[key]) {
+          console.error("Missing translation for " + key);
+        } else {
+          //If translation applies to a specific attr, set it, else set innerHTML
+          if (STRINGS[key].attr) {
+            el.setAttribute(STRINGS[key].attr, STRINGS[key][lang]);
+          } else {
+            el.innerHTML = STRINGS[key][lang];
+          }
+        }
+      });
```

Comment:
I skipped over this the first time around.  Very nice extension of this module to handle attributes ❤️